074-additional-volumes-support.md

```diff
@@ -0,0 +1,138 @@
+
+|Pod              |CRD              |Schema Location                                                                       |Implementing class           |
+|-----------------|-----------------|--------------------------------------------------------------------------------------|-----------------------------|
+|Kafka            |Kafka            |spec -> kafka -> template -> pod/kafkaContainer/initContainer                         |KafkaCluster.java            |
+|                 |KafkaNodePoolSpec|spec -> template -> pod/kafkaContainer/initContainer                                  |                             |
+|Zookeeper        |Kafka            |spec -> zookeeper -> template -> pod/zookeeperContainer                               |ZookeeperCluster.java        |
+|EntityOperator   |Kafka            |spec -> entityOperator -> template -> pod/topicOperatorContainer/userOperatorContainer|EntityOperator.java          |
+|CruiseControl    |Kafka            |spec -> cruiseControl -> template -> pod/cruiseControlContainer                       |CruiseControl.java           |
+|KafkaExporter    |Kafka            |spec -> kafkaExporter -> template -> pod/container                                    |KafkaExporter.java           |
+|KafkaConnect     |KafkaConnect     |spec -> template -> pod/connectContainer/initContainer/buildContainer                 |KafkaConnectCluster.java     |
+|KafkaBridge      |KafkaBridge      |spec -> template -> pod/bridgeContainer/initContainer                                 |KafkaBridgeCluster.java      |
+|KafkaMirrorMaker2|KafkaMirrorMaker |spec -> template -> pod/connectContainer/initContainer/buildContainer                 |KafkaMirrorMaker2Cluster.java|
+
+
+It is proposed to not add support in the following locations - if `volumes` or `volumeMounts` are specified in the PodTemplate or ContainerTemplate in any of these locations it will be ignored:
+
+|Pod             |CRD             |Schema Location                                          |Reason                                 |
+|----------------|----------------|---------------------------------------------------------|---------------------------------------|
+|KafkaMirrorMaker|KafkaMirrorMaker|spec -> template -> pod/mirrorMakerContainer             |KafkaMirrorMaker has been deprecated   |
+|Kafka           |Kafka           |spec -> jmxTrans -> template -> pod/container            |JmxTrans no longer supported           |
+|EntityOperator  |Kafka           |spec -> entityOperator -> template -> tlsSidecarContainer|tlsSidecarContainer no longer supported|
+|CruiseControl   |Kafka           |spec -> cruiseControl -> template -> tlsSidecarContainer |tlsSidecarContainer deprecated         |
+
+
+An example configuration could look as follows:
+
+```yaml

```

Comment:
Can you update the example to match the other changes? E.g. the `/mnt/...` muont paths, the volume types supported etc.